keyboards/blank_tehnologii/manibus/keymaps/default/keymap.c

```diff
@@ -57,7 +61,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 //   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
       KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                            KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_BSLS,
 //   ├────────┼────────┼────────┼────────┼────────┼────────┼────────┐     ┌────────┼────────┼────────┼────────┼────────┼────────┼────────┤
-      KC_LCTL, KC_HOME, KC_INS,  KC_DEL,  KC_LGUI, KC_SPC,  KC_LALT,       KC_ENT,  KC_BSPC, LOWER,   KC_LBRC,  KC_RBRC,  KC_END,  KC_GRV
+      KC_LCTL, KC_HOME, KC_INS,  KC_DEL,  KC_LGUI, KC_SPC,  KC_LALT,       KC_ENT,  KC_BSPC, MO(LOWER),KC_LBRC,KC_RBRC,  KC_END,  KC_GRV
```

Comment:
`MO(LOWER)` wont work as its a custom keycode, as such, the changes to this file should be reverted.

Reply:
Yes you're right, I got confused and thought I had an issue with the default keymap but it was actually fine before.